src/panels/config/helpers/forms/ha-input_timetable-form.ts

```diff
@@ -0,0 +1,122 @@
+import "@polymer/paper-input/paper-input";
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  internalProperty,
```

Comment:
`internalProperty` was renamed to `state` in LitElement
```suggestion
  state,
```